discussment-core/src/main/java/org/danekja/discussment/core/exception/MessageLengthExceeded.java

```diff
@@ -0,0 +1,15 @@
+package org.danekja.discussment.core.exception;
+
+/**
+ * Exception thrown when trying to save post that is too long.
+ */
+public class MessageLengthExceeded extends Exception {
+
+    public MessageLengthExceeded(int messageLength, int limit) {
+        super("Message length ("+messageLength+") exceeds the limit ("+limit+")");
```

Comment:
Not crucial at all but I like to use `String.format` instead of join strings by +.